lib/compiler/src/beam_split.erl

```diff
@@ -56,9 +56,6 @@ split_block([{set,[D],[S|Puts],{alloc,R,{put_map,Op,{f,Lbl}=Fail}}}|Is],
 	    Bl, Acc) when Lbl =/= 0 ->
     split_block(Is, [], [{put_map,Fail,Op,S,D,R,{list,Puts}}|
 			 make_block(Bl, Acc)]);
-split_block([{set,Ds,[S|Ss],{get_map_elements,Fail}}|Is], Bl, Acc) ->
-    Gets = beam_utils:join_even(Ss,Ds),
```

Comment:
I think this is the only occurrence of `beam_utils:join_even/2`? Do you want to remove the implementation from `beam_utils`. It was specifically designed for handling register lists for maps.